istioctl/cmd/authz.go

```diff
@@ -45,17 +44,21 @@ var (
 var (
 	checkCmd = &cobra.Command{
 		Use:   "check <pod-name>[.<pod-namespace>]",
-		Short: "Check the authorization configuration based on Envoy config",
-		Long: `Check analyzes the authorization configuration based on the Envoy config. The Envoy config could
-be provided either by pod name or from a config dump file (the whole output of http://localhost:15000/config_dump
-of an Envoy instance).
+		Short: "Check Envoy config dump for filters configured for access control",
+		Long: `Check Envoy config dump for filters configured for access control.
+The command reads the Envoy config dump for the given pod and checks the filter
+configuration related to access control. For example, it shows whether or not the
+Envoy is configured with a RBAC filter and the configured rule names if there is one.
+
+The Envoy config could be provided either by pod name or from a config dump file
+(the whole output of http://localhost:15000/config_dump of an Envoy instance).
 
 THIS COMMAND IS STILL UNDER ACTIVE DEVELOPMENT AND NOT READY FOR PRODUCTION USE.
 `,
-		Example: `  # Check the authorization configuration for pod httpbin-88ddbcfdd-nt5jb in namespace foo:
+		Example: `  # Check the Envoy config dump from the pod httpbin-88ddbcfdd-nt5jb in namespace foo:
```

Comment:
Why changed from `authorization configuration` to `Envoy config dump`? Wouldn't it better to be more specific that this is checking Envoy authz configuration?

Reply:
Updated.